.changeset/entries/2021-07-16T07:52:37Z.yaml

```diff
@@ -0,0 +1,5 @@
+type: added
+module: x/profiles
+pull_request: 562
+description: Added expiration for application links
+backward_compatible: true
```

Comment:
This is definitely not backward compatible as it adds a new feature that requires parameters changes etc